README.md

```diff
@@ -68,34 +68,49 @@ import dame_flame
 result = dame_flame.DAME_FLAME.DAME(input_data=df, treatment_column_name="treated", outcome_column_name="outcome", holdout_data=1.0)
 print(result[0])
-#>    x1   x2   x3   x4
-#> 0   0   1    1    *
-#> 1   0   1    1    *
-#> 2   1   0    *    *
-#> 3   1   0    *    *
+#>    x1   x2   x3   x4    weights
+#> 0   0   1    1    *     1
+#> 1   0   1    1    *     1
+#> 2   1   0    *    1     1
+#> 3   1   0    *    1     1
+
+print(result[1])
+#> [[0,1],[2,3]]
 ```
-result is a list, where the first element in the list is of type **Data Frame**. The dataframe contains all of the units that were matched, and the covariates and corresponding values, that it was matched on. The covariates that each unit was not matched on is denoted with a " * " character. The list 'result' will have additional values based on additional optional parameters, detailed in additional documentation below. 
+result is a list, where the first element in the list is of type **Data Frame**. The dataframe contains all of the units that were matched, and the covariates and corresponding values, that it was matched on. The covariates that each unit was not matched on is denoted with a " * " character. This element also includes a column of unit weights which specifies the number of times each unit was matched. The second element is a list of unit IDs belonging to each of the matched groups created by the algorithm. The list 'result' will have additional values based on additional optional parameters, detailed in additional documentation below. 
 
-To find the main matched group of a particular unit after DAME has been run, use the function *mmg_of_unit*
+To find the main matched group of a particular unit or group of units after DAME has been run, use the function *MG*:
 
 ```Python
-mmg = dame_flame.DAME_FLAME.mmg_of_unit(return_df=result[0], unit_id=0, input_data=df)
+mmg = dame_flame.DAME_FLAME.MG(return_df=result, unit_id=0, input_data=df)
 print(mmg)
 
-#>    x1   x2    x3
-#> 0   0    1    1
-#> 1   0    1    1
+#>      x1    x2    x3    x4    treated    outcome
+#> 0    0     1     1     *     0          5
+#> 1    0     1     1     *     1          6
 ```
 
-To find the treatment effect of a unit, use the function *te_of_unit*
+To find the conditional treatment effect (CATE) for the main matched group of a particular unit or group of units, use the function *CATE*:
 
 
 ```Python
-te = dame_flame.DAME_FLAME.te_of_unit(return_df=result[0], unit_id=2, input_data=df, treatment_column_name='treated', outcome_column_name='outcome')
+te = dame_flame.DAME_FLAME.CATE(return_df=result, unit_id=2, input_data=df)
 print(te)
-#> -1.0
+#> 3.0
 ```
 
+To find the average treatment effect (ATE) or average treatment effect on the treated (ATT), use the functions *ATE* and *ATT*, respectively:
+
+
+```Python
+ate = dame_flame.DAME_FLAME.ATE(return_df=result, input_data=df)
+print(ate)
+#> 2.0
+
+att = dame_flame.DAME_FLAME.ATT(return_df=result, input_data=df)
+print(att)
+#> 2.0
+```
```

Comment:
Could combine these into one codeblock/comment block if you wanted? Up to how you think it looks better, just pointing out that you have the option :)

Reply:
Do you mean something like this?

```Python
ate = dame_flame.DAME_FLAME.ATE(return_df=result, input_data=df)
att = dame_flame.DAME_FLAME.ATT(return_df=result, input_data=df)
print(ate, att)
#> 2.0  2.0
```

Reply:
I meant that you go:
- text description of cate
- code block of cate
- text description of att/ate
- code block of att/ate

so if you wanted you could go
- text description of cate/att/ate
- code block of all 3

or 6 alternating text + code blocks and separate att/ate...?

up to you whatever you think looks best aesthetically, I'm fine with whatever